### ISSUE TITLE
[Accessibility][HC] The color of the inverted triangle symbol of "Details" button in Properties window is incorrect when non-selecting it

### ISSUE THREAD

**Issue by MelonWang1** [dotnet/winforms#7608]
### .NET version

.NET SDK 7.0.100-rc.2.22417.1

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, it's not a regression issue, also repro on .NET 6.0.

### Issue description

**Current Behavior:**
The inverted triangle symbol of "Details" button is invisible when non-selecting it.
![issue36](https://user-images.githubusercontent.com/94418985/185284204-c3b648c8-0e4e-4b5e-a7e4-177366875041.png)

**Expected Behavior:**
The inverted triangle symbol of "Details" button should be visible when non-selecting it.
![issue36-expectr](https://user-images.githubusercontent.com/94418985/185284218-771cc459-932d-42dd-89c2-5f96fc6097c1.png)
**More Info:**
1. This issue is repro on theme is HC1/2/Black.
2. This issue is not repro on theme White.

### Steps to reproduce

1. Create a WinForms .NET core project
2. Change the Form's Name properties to ";"
3. Now "Properties Window" will pop up
4. Observe the inverted triangle symbol of "Details" button

**Comment by dmitrii-drobotov** [comment]
The icon was changed in Core and now has sufficient contrast ratio in different HC themes due to an added border:
![image](https://user-images.githubusercontent.com/102954094/199940558-a1e4503e-6e0f-4976-8025-71b0b2bac3e2.png)
![image](https://user-images.githubusercontent.com/102954094/199943657-8dd7e576-9682-4b78-87e5-cf8377bfce0a.png)
![image](https://user-images.githubusercontent.com/102954094/199943814-59dd7373-e962-46b6-94df-80b5a31c589e.png)
![image](https://user-images.githubusercontent.com/102954094/199943875-0630cc66-0e8d-479f-accd-402287539eb8.png)

These screenshots are from PropertyGrid in a regular .NET 7 WinForms app, but in 4.8.1 and VS' Properties window it still remains an issue, as in screenshots in description. @merriemcgaw is there anything we should do now or we can close this issue?

**Comment by dmitrii-drobotov** [comment]
Closing as it's resolved in runtime.

**Comment by RussKie** [comment]
> Closing as it's resolved in runtime.

Was it fixed in dotnet/runtime? If so, do you have a link to that work item?

**Comment by dmitrii-drobotov** [comment]
> > Closing as it's resolved in runtime.
> 
> Was it fixed in dotnet/runtime? If so, do you have a link to that work item?

No, I mean the winforms runtime, as opposed to the winforms designer, where this issue was observed.
